api/src/main/java/com/codeforcommunity/dto/imports/TreeSpeciesImport.java

```diff
@@ -13,7 +13,8 @@ public class TreeSpeciesImport extends ApiDto {
 
   private String defaultImage;
 
-  public TreeSpeciesImport(String genus, String species, String commonName, String speciesCode, String defaultImage) {
+  public TreeSpeciesImport(
+      String genus, String species, String commonName, String speciesCode, String defaultImage) {
```

Comment:
❓ Is there a way to limit the autoformatter to specific files? It's definitely nice to have our code look good, but a bunch of extra files end up in the PR, and it can be hard to understand what the actual changes made are vs. just format changes

Reply:
do you think the solution would be to try to only add the files we meant to change before committing?

Reply:
> do you think the solution would be to try to only add the files we meant to change before committing?

Yeah that would work too - just make sure to keep note of what files you changed, so you don't forget to add all the files you worked on